pkg/controller/provider/container/ovirt/reconciler.go

```diff
@@ -422,23 +411,14 @@ func (r *Reconciler) apply(changeSet []Updater) (err error) {
 
 //
 // List Event collection.
-// Query by list of event types and date.
-func (r *Reconciler) listEvent() (itr fb.Iterator, err error) {
+// Query by list of event types since lastEvent (marked).
+func (r *Reconciler) listEvent() (list []Event, err error) {
```

Comment:
Events are small and are already in memory so not much (anything) gained by using the iterator.